modules/src/core/ics04_channel/packet.rs

```diff
@@ -110,6 +110,46 @@ pub struct Packet {
     pub timeout_timestamp: Timestamp,
 }
 
+struct PacketData<'a>(&'a [u8]);
+
+impl<'a> ::core::fmt::Debug for PacketData<'a> {
+    fn fmt(&self, formatter: &mut ::core::fmt::Formatter<'_>) -> Result<(), ::core::fmt::Error> {
```

Comment:
Minor nit: it would be better to use `core::fmt` at the top to reduce namespacing cruft here.

Reply:
I usually like to be explicit about path resolution; this version makes its clear when quickly glancing at it that I'm referring to the `core` crate rather than a potential `core` submodule (which in the general case I have to go check if such a module exists). However it's very minor indeed, and removing it has the benefit of conforming to the style of the rest of the repo.

Come to think of it, it's also by and large mitigated by being careful not to name any of our modules the name of an external crate.